proto/btc/get_xpubs.proto

```diff
@@ -26,13 +25,17 @@ message GetXpubsIntiateRequest {
   repeated GetXpubDerivationPath derivation_paths = 2;
 }
 
+message GetXpubsFetchNextRequest {
+}
+
```

Comment:
Do add comments on how this flow should work. For reference check comments on `sign_txn`

Reply:
added